scripts/entities_csv2json.py

```diff
@@ -93,6 +92,8 @@
                         [x.strip().lower() for x in rawSynonyms]
                         representativeValue = rawSynonyms[0]
                         synonyms = sorted(list(set(rawSynonyms[1:])))
+                        # remove value from synonyms, so that duplicity with value is not possible
+                        synonyms.remove(representativeValue)
```

Comment:
I am not sure what happens if removing not existing value. Can you please test @Tereza-Pytelova

Reply:
Tested and hopefully fixed.